docs/en/developer.md

```diff
@@ -423,6 +423,38 @@ for the advanced settings.
 
 ## Advanced features
 
+### Allowing email login on LDAP login form
+
+If you have registered `LDAPAuthenticator` (see "Configure SilverStripe Authenticators" section), you may notice the only
```

Comment:
I'd probably replace this paragraph by saying *why* this feature is not enabled by default.